pkg/serialization/codec/jam/decode_test.go

```diff
@@ -0,0 +1,25 @@
+package jam
+
+import (
+	"bytes"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestDecodeBits(t *testing.T) {
+	input := []byte{17, 25, 0, 1, 2}
```

Comment:
can we add edge cases? single bit case or bit sequences not multiple of 8 (eg 10 bits)